### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by sedrubal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
-prerequisites:
+setup
+-----
 
-```sudo pip install oerplib```
-```apt-get install python-repoze.lru python-natsort```
+```sh
+# If you prefer apt packages:
+apt-get install python-repoze.lru python-natsort
```
Comment (mgmax) [inline README.md:11]: you still need to install oerplib, it doesn't come via apt.

Comment (sedrubal) [inline README.md:11]: It's here: https://github.com/fau-fablab/oerp-pricelist/blob/ba903a962352e3fedcbbb5716735303f3fa2ef17/requirements.txt#L1

Comment (mgmax) [inline README.md:11]: then the comment "otherwise" [deutsch: ansonsten] is misleading

Comment (sedrubal) [inline README.md:11]: Oh right, I didn't thought of this...

**Pull request merged**